src/PayU/PaymentsApi/AluV3/AluV3.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+
+namespace PayU\PaymentsApi\AluV3;
+
+use PayU\Alu\Exceptions\ConnectionException;
+use PayU\Alu\HashService;
+use PayU\Alu\HTTPClient;
+use PayU\Alu\Request;
+use PayU\PaymentsApi\AluV3\Exceptions\ResponseBuilderException;
+use PayU\PaymentsApi\AluV3\Exceptions\ResponseParserException;
+use PayU\PaymentsApi\AluV3\Services\RequestBuilder;
+use PayU\PaymentsApi\AluV3\Services\ResponseBuilder;
+use PayU\PaymentsApi\AluV3\Services\ResponseParser;
+use PayU\PaymentsApi\Exceptions\AuthorizationException;
+use PayU\PaymentsApi\Interfaces\AuthorizationInterface;
+
+final class AluV3 implements AuthorizationInterface
+{
+    const ALU_URL_PATH = '/order/alu/v3';
+    const API_VERSION_V3 = "v3";
+
+    /**
+     * @var array
+     * todo set ro to original value
+     */
+    private $aluUrlHostname = [
+        'ro' => 'https://secure.payu.ro',
+        'ru' => 'https://secure.payu.ru',
+        'ua' => 'https://secure.payu.ua',
+        'hu' => 'https://secure.payu.hu',
+        'tr' => 'https://secure.payu.com.tr',
+    ];
+
+    /**
+     * @var HTTPClient
+     */
+    private $httpClient;
+
+    /**
+     * @var HashService
+     */
+    private $hashService;
+
+    /**
+     * @var RequestBuilder
+     */
+    private $requestBuilder;
+
+    /**
+     * @var ResponseBuilder
+     */
+    private $responseBuilder;
+
+    /**
+     * @var ResponseParser
+     */
+    private $responseParser;
+
+    public function __construct(
+        HTTPClient $httpClient,
```

Comment:
Since we marked HTTPClient and HashService  as deprecated, we should use the new classes instead of the old ones